src/fragments/lib/auth/common/mfa/update_mfa.mdx

```diff
@@ -0,0 +1,48 @@
+As mentioned above, MFA cannot be unconditionally enabled for all users after creating a user pool. The following steps show how to enable MFA as "Optional" for users. In this mode, MFA must be enabled on a user-by-user basis, either through an Admin SDK (e.g. via a Lambda trigger as part of the sign-up process), or manually in the Cognito console.
```

Comment:
we can remove "As mentioned above, MFA cannot be unconditionally enabled for all users after creating a user pool"

Reply:
For the rest of the section: 

When updating your Auth resources, you can set up MFA to be set to either "Optional" or "Off".